bos

```diff
@@ -1014,6 +1015,7 @@ prog
           logger,
           after: new Date(Date.now() - (1000 * 60 * 60 * 6)).toISOString(),
           ask: (n, cbk) => inquirer.prompt([n]).then(res => cbk(res)),
+          cooperative_close_addresses: options.coopCloseAddress || undefined,
```

Comment:
if this just takes 1 address, shouldn't it be called `cooperative_close_address`

Reply:
fixed it.